browser-checklist.md

```diff
@@ -0,0 +1,26 @@
+## Browsers
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] FF
+- [ ] IE10
+- [ ] IE9
+- [ ] IE8
+- [ ] IE7
+- [ ] Opera
+- [ ] iOS8
+- [ ] Android 5
```

Comment:
SauceLabs provides Android 4 through 5.1, which I presume is the Android browser, which is based on Blink—the same engine Chrome for Android uses. I'm not sure what the roadmap is for having both on Android. I'm guessing maybe Chrome will be the default at some point? http://caniuse.com/ lists both, but that may be to cover legacy support, as opposed to feature-support differences. I handled a Samsung Galaxy Tab recently that came with both installed out of the box.

IIRC I was not able to get SauceConnect to work with SauceLab's Android VM to test local changes.

The other option is to install the [Android emulator](http://developer.android.com/tools/help/emulator.html) locally or in a VM, or have an actual Android device available on a local network and test via Ghostlab. 

Chrome device emulation is probably fine, particularly if Chrome for Android is the way things are going.